django_project/changes/management/commands/purge_unused_images.py

```diff
@@ -32,7 +32,7 @@ def handle(self, *args, **options):
                 self.stdout.write('Moving %s' % image_path)
             image_file = File(image_path)
             entry.image_file.save(image_path, image_file, True)
-            #os.remove(image_path)
+            # os.remove(image_path)
```

Comment:
I wonder why this is commented out...maybe I was debugging and accidentally forgot to remove the # - can you check?